### ISSUE TITLE
Chart not rendered

### ISSUE THREAD

**Issue by truckee** [symfony/ux#296]
Although the code inspection shows the chart seems to be properly defined, no chart is rendered. It's conceivable I missed something, but what?

Pieces of the puzzle:
Symfony 6.0.6

composer: 
`symfony/ux-chartjs`: `v2.1.0`

packages.json:
"@hotwired/stimulus": "^3.0.1"
"@symfony/stimulus-bridge": "^3.0.0"
"@symfony/webpack-encore": "^1.7.0"

`bootstrap.js`:
```
import { startStimulusApp } from '@symfony/stimulus-bridge';

// Registers Stimulus controllers from controllers.json and in the controllers/ directory
export const app = startStimulusApp(require.context(
    '@symfony/stimulus-bridge/lazy-controller-loader!./controllers',
    true,
    /\.[jt]sx?$/
));
```

`app.js` includes:
```import { startStimulusApp } from '@symfony/stimulus-bridge';

export const app = startStimulusApp(require.context(
    '@symfony/stimulus-bridge/lazy-controller-loader!./controllers',
    true,
    /\.(j|t)sx?$/
));
```

Admin controller:
```
    public function index(ChartService $charter, ChartBuilderInterface $chartBuilder)
    {
        $v_chart = $charter->volunteerChart();
        $volChart = $chartBuilder->createChart(Chart::TYPE_LINE);
        $volChart->setData($v_chart['data']);
        $volChart->setOptions($v_chart['options']);
        ....

        return $this->render('Admin/index.html.twig', [
                    'vol_chart' => $volChart,
                    ...
        ]);
    }
```
where ChartService simply generates the data and options to be applied.

template: `{{ render_chart(vol_chart) }}`

Inspector's HTML:
```
<canvas data-controller="symfony--ux-chartjs--chart" data-symfony--ux-chartjs--chart-view-value="{"type":"line","data":{"labels":["Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec","Jan","Feb"],"datasets":[{"label":"Volunteer registrations by month","backgroundColor":"rgb(255, 99, 132)","borderColor":"rgb(255, 99, 132)","data":[4,9,16,23,29,34,38,41,44,46,47,48]}]},"options":{"scales":{"y":{"suggestedMin":0,"suggestedMax":100}}}}"></canvas>
```

Edit: more readable HTML

**Comment by truckee** [comment]
In an effort to avoid any extraneous mistakes I distilled the code from a current project to a [repo at github](https://github.com/truckee/chart_test).  If you run it with `symfony serve -d`, just launching `127.0.0.1:8000` will take you to a blank page where the page source code contains the rendered `<canvas>...</canvas>`.

The setup for the distilled code was simply:
```
symfony new chart_test --webapp
composer require symfony/ux-chartjs
yarn install --force
yarn encore dev
```
I then added three files: a controller, a service and a template.
